[link-grammar] Python bindings

In short I had to add `ncurses` package as it was listed in python extra libraries.

This is the error:
```
  Could not link test program to Python. Maybe the main Python library has been
  installed in some non-standard library path. If so, pass it to configure,
  via the LIBS environment variable.
  Example: ./configure LIBS="-L/usr/non-standard-path/python/lib"
  ============================================================================
   ERROR!
   You probably have to install the development version of the Python package
   for your distribution.  The exact name of this package varies among them.
  ============================================================================

See `config.log' for more details
```
And from `config.log`:
```
configure:20241: checking for python
configure:20259: found /nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/bin/python
configure:20271: result: /nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/bin/python
configure:20289: checking for a version of Python >= '2.1.0'
configure:20326: result: yes
configure:20334: checking for a version of Python >= '2.6'
configure:20340: result: yes
configure:20363: checking for the distutils Python package
configure:20367: result: yes
configure:20385: checking for Python include path
configure:20401: result: -I/nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/include/python3.6m
configure:20408: checking for Python library path
configure:20490: result: -L/nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/lib -lpython3.6m
configure:20497: checking for Python site-packages path
configure:20503: result: /nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/lib/python3.6/site-packages
configure:20510: checking python extra libraries
configure:20517: result: -lpthread -ldl -lcrypt -lncurses -lutil -lm
configure:20524: checking python extra linking flags
configure:20531: result: -Xlinker -export-dynamic
configure:20538: checking consistency of all components of python development environment
configure:20566: gcc -o conftest -D_DEFAULT_SOURCE -std=c11 -D_BSD_SOURCE -D_SVID_SOURCE -D_GNU_SOURCE -D_ISOC11_SOURCE -g -O2 -O3  -I/nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/include/python3.6m  -Xlinker -export-dynamic conftest.c  -L/nix/store/80qva2zal82njkg4zvb7xg94mnikqksd-python3-3.6.8/lib -lpython3.6m -lpthread -ldl -lcrypt -lncurses -lutil -lm -lpthread -ldl -lcrypt -lncurses -lutil -lm >&5
/nix/store/5vyv136pqs75pj0b8vcpdyc03dmn9p0n-binutils-2.30/bin/ld: cannot find -lncurses
collect2: error: ld returned 1 exit status
```

--- a/packages/link-grammar.nix
+++ b/packages/link-grammar.nix
@@ -22,6 +22,11 @@ stdenv.mkDerivation rec {
     flex
     graphviz
     zlib
+
+    python3
+
+    ncurses # needed for python bindings..
+    sqlite
   ];
 
   # fix for https://github.com/NixOS/nixpkgs/issues/38991

